From 3b1d5d60b0578d84d4cb9ea629f57388b1281eb2 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 22 Dec 2015 14:15:40 +0800
Subject: [PATCH 1260/1691] MLK-12059 ARM: dts: imx6ul-14x14-evk: add mic
 detect gpio to support headset Jack

commit 3b1d5d60b0578d84d4cb9ea629f57388b1281eb2 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

imx6ul-evk using headset jack, so mic jack and headphone jack should
using the same gpio for detect.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
(cherry picked from commit 99eff3b520447a119f1ddc62af29f40b1acdeebb)
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dts | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dts b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
index 52cac9f..0718a6d 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
@@ -89,18 +89,19 @@
 		 */
 		hp-det = <3 0>;
 		hp-det-gpios = <&gpio5 4 0>;
+		mic-det-gpios = <&gpio5 4 0>;
 		audio-routing =
-			"Headset Jack", "HP_L",
-			"Headset Jack", "HP_R",
+			"Headphone Jack", "HP_L",
+			"Headphone Jack", "HP_R",
 			"Ext Spk", "SPK_LP",
 			"Ext Spk", "SPK_LN",
 			"Ext Spk", "SPK_RP",
 			"Ext Spk", "SPK_RN",
-			"LINPUT2", "Hp MIC",
-			"LINPUT3", "Hp MIC",
+			"LINPUT2", "Mic Jack",
+			"LINPUT3", "Mic Jack",
 			"RINPUT1", "Main MIC",
 			"RINPUT2", "Main MIC",
-			"Hp MIC", "MICB",
+			"Mic Jack", "MICB",
 			"Main MIC", "MICB",
 			"CPU-Playback", "ASRC-Playback",
 			"Playback", "CPU-Playback",
-- 
1.9.1

